brigade-cr-gateway/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.6
+
+RUN apk update && apk add --no-cache \
+    ca-certificates \
+    git \
+    && update-ca-certificates
+
+COPY rootfs /
+
+ENV GIT_SSH=/gitssh.sh
+
+CMD /usr/bin/brigade-cr-gateway
```

Comment:
binary was copied to `/` https://github.com/Azure/brigade/blob/master/brigade-gateway/Dockerfile

Reply:
Weird... I see it in /usr/bin/brigade-cr-gateway

Reply:
Ah... rebasing mistake maybe?